app/models/tournament_report.rb

```diff
@@ -31,10 +31,12 @@ def player_report(player)
 
     # The following is dedicated to Winter Zkqxj
     first, *rest = player.name.split(" ")
+    last = player.name.split(" ").last
```

Comment:
this should work:
```
first_names, _, last = player.name.rpartition(" ")
```

rpartition starts from the right so it will split on the last space